src/Cover/DefaultPdfGenerator.php

```diff
@@ -172,6 +179,32 @@ public function setLicenceLogosDir($licenceLogosDir)
         $this->licenceLogosDir = $licenceLogosDir;
     }
 
+    /**
+     * Returns the list of Config option keys whose values will be made available as metadata during PDF generation.
+     *
+     * @return string[] List of Config option keys
+     */
+    public function getConfigOptionKeys()
+    {
+        $configOptionKeys = $this->configOptionKeys;
+
+        if (empty($configOptionKeys)) {
+            $configOptionKeys = ['name', 'url'];
+        }
+
+        return $configOptionKeys;
+    }
+
+    /**
+     * Sets the list of Config option keys whose values shall be made available as metadata during PDF generation.
+     *
+     * @param string[] $configOptionKeys
+     */
+    public function setConfigOptionKeys(array $configOptionKeys): void
```

Comment:
Bitte "array" weglassen. Bislang verwenden wir @param, es sei den wir erben von externem Code. In jedem Fall sollten die Deklarationen nicht unterschiedlich sein.